### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ashumz**
Updates to clarify how the docs are previewed and deployed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,8 @@
 
 This repository contains the documentation source files for [coder.com/docs](https://coder.com/docs).
 
-Pre-production documentation is drafted in a private repository before being
-synchronized with this repository as releases are made public.
+Is something confusing, unclear, or just want to contribute something new? [Open an issue or pull request](https://github.com/cdr/docs/issues/new)! While you can't currently build the docs site locally, you'll be able to preview your changes in the context of the site through a deploy preview. 
 
-Interested in using Coder? [Contact our Technical Sales Team](https://coder.com/contact).
+Our documentation is published once a day, so merged pull requests may not be visible right away but will be visible within 24 hours.
```
Comment (nathanpotter) [inline README.md:7]:
```suggestion
Our documentation is published once a day, so merged pull requests may not be visible right away, but will be visible within 24 hours.
```

File: README.md [hunk 2]
```diff
@@ -2,9 +2,8 @@
 
 This repository contains the documentation source files for [coder.com/docs](https://coder.com/docs).
 
-Pre-production documentation is drafted in a private repository before being
-synchronized with this repository as releases are made public.
+Is something confusing, unclear, or just want to contribute something new? [Open an issue or pull request](https://github.com/cdr/docs/issues/new)! While you can't currently build the docs site locally, you'll be able to preview your changes in the context of the site through a deploy preview.
```
Comment (nathanpotter) [inline README.md:5]: these lines are a bit long, would recommend a space at 80 chars to follow markdown convention

File: README.md [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
 
-Interested in using Coder? [Contact our Technical Sales Team](https://coder.com/contact).
+Contributions can be previewed in the context of the site using pull request
+deploy previews. :rocket:
 
-See something wrong or unclear? Please contribute by [opening an issue or pull request](https://github.com/cdr/docs/issues/new).
+Our documentation is published once a day. Expect any merged contributions to
+be live within 24 hours.
+
+**Interested in using Coder?** [Contact our Technical Sales Team](https://coder.com/contact).
```
Comment (vapurrmaid) [inline README.md:15]: Thoughts on the following?

# Coder Docs

This repository contains the documentation source files for
[coder.com/docs](https://coder.com/docs).

## Contributing

Is something confusing, unclear, or just want to contribute something new?
[Open an issue or pull request](https://github.com/cdr/docs/issues/new)!

Contributions can be previewed in the context of the site using pull request
deploy previews. :rocket:

Our documentation is published once a day. Expect any merged contributions to
be live within 24 hours.

## Interested in using Coder?

[Contact our Technical Sales Team](https://coder.com/contact).

Comment (ashumz) [reply README.md:15]: oh yeah, that's good, i was just working with the original format trying to make contribution situation slightly more clear :) will add that.

**Review by vapurrmaid — APPROVED** [review]
This looks awesome to me :tada:

**Pull request merged**